images/saige-qtl/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-r:1.4
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+RUN micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
+        -c cpg -c bioconda -c conda-forge -c anaconda \
+        r-essentials r-base=4.1.2 python=3.10 \
+        cmake gettext lapack r-matrix \
+        r r-rcpp r-rcpparmadillo r-data.table r-bh r-matrix \
```

Comment:
`r` is again just the channel. You'll want to move all the `-c` parameters to the top.